.rubocop.yml

```diff
@@ -1,6 +1,9 @@
 require:
   - rubocop-performance
 
+AllCops:
+  TargetRubyVersion: 2.2
+
```

Comment:
Given how many cops I had to turn *off* in this PR, it might be better to disable them all here and explicitly turn on the ones you want. I’m not sure which you want, though, so I just turned off all the new and/or error-triggering ones.